packages/manager/apps/pci-kubernetes/src/api/hooks/instances.spec.ts

```diff
@@ -0,0 +1,70 @@
+import { renderHook, waitFor } from '@testing-library/react';
+import { describe, it, vi } from 'vitest';
+import * as ApiInstanceModule from '@/api/data/instance';
+import { useInstances, useSwitchToMonthlyBilling } from '@/api/hooks/instances';
+import { wrapper } from '@/wrapperRenders';
+
+vi.mock('@/api/data/instance', async () => {
+  const mod = await vi.importActual('@/api/data/instance');
+  return {
+    ...mod,
+    getInstances: vi.fn(),
+    switchToMonthlyBilling: vi.fn(),
+  };
+});
```

Comment:
You dont need the mock of the module here, since you spyOn the needed functions bellow